Frontend/src/packages/dashboard/Dashboard.tsx

```diff
@@ -7,33 +7,34 @@ import SideBar from 'src/packages/dashboard/components/SideBar/SideBar';
 import Renderer from './components/FileRenderer/Renderer';
 import { initAction } from './state/folders/actions';
 import ConfirmationWindow from './components/ConfirmationModal/ConfirmationWindow';
-import { Folder, File } from './state/folders/types';
+import Directory from "./components/Directory";
 
 
 const Container = styled.div`
   display: flex;
   flex-direction: row;
 `;
 
-export default function Dashboard() {
+export default function Dashboard(this: any) {
```

Comment:
what is `this: any`? `this` is usually not a very good variable name due to the conflicts with OOP standards

Reply:
Hmmm, I'm not really sure, my IDE must added this and I didn't pay attention to the changes.